example/example/templates/_base.html

```diff
@@ -17,9 +17,11 @@
 </head>
 
 <!--
-  Enable htmx extensions for all requests:
-  * debug: https://htmx.org/extensions/debug/
-  * event-header: https://htmx.org/extensions/event-header/
+  Enable htmx extensions for all requests - depending on HTMX Version:
+  * debug v1 HTMX: https://v1.htmx.org/extensions/debug/
+  * debug v2 HTMX: https://github.com/bigskysoftware/htmx-extensions/blob/main/src/debug/README.md
+  * event-header for v1 HTMX: https://v1.htmx.org/extensions/event-header/
+  * event-header for v2 HTMX: https://github.com/bigskysoftware/htmx-extensions/blob/main/src/event-header/README.md
```

Comment:
no need to link to v1 AND v2, we can just use v2 in the example project

```suggestion
  Enable htmx extensions for all requests:
  * debug: https://github.com/bigskysoftware/htmx-extensions/blob/main/src/debug/README.md
  * event-header: https://github.com/bigskysoftware/htmx-extensions/blob/main/src/event-header/README.md
```